pkgs/jni/lib/src/errors.dart

```diff
@@ -100,9 +102,29 @@ final class HelperNotFoundError extends Error {
 
   @override
   String toString() => '''
-Lookup for helper library $path failed.
-Please ensure that `dartjni` shared library is built.
-Provided jni:setup script can be used to build the shared library.
-If the library is already built, ensure that the JVM libraries can be 
-loaded from Dart.''';
+Unable to locate the helper library.
+Ensure that the helper library is available at the path: $path.
+Use the provided jni:setup script to generate the shared library if it does not exist.
+If the library is already built, you may need to explicitly call Jni.spawn with the dylibDir argument set to the directory containing the shared library.
+The directory containing the shared library is typically in build\\jni_libs.
+''';
+}
+
+final class DynamicLibraryLoadError extends Error {
+  final String libraryPath;
+
+  DynamicLibraryLoadError(this.libraryPath);
+
+  @override
+  String toString() {
+    const windowsErrorMessage = r'that `\bin\server\jvm.dll` is in the PATH.';
```

Comment:
This will be inserted at the end, but it's not a full sentence. 

```
...
Please ensure that all dependencies are installed and that the file has the correct permissions.
that `\bin\server\jvm.dll` is in the PATH.
```

Reply:
Good catch.

Updated to be:

Mac & Linux:
```
Please ensure that the file has the correct permissions.
```

Windows:
```
Please ensure that `\bin\server\jvm.dll` is in the PATH, and that the file has the correct permissions.
```